### PR TITLE
Added 'Learn Windows PowerShell in a Month of Lunches' as recommended reading

### PR DESCRIPTION
**Pull request by tobvil**
Added one of the most popular PowerShell books 'Learn Windows PowerShell in a Month of Lunches' by Donald W. Jones and Jeffrey Hicks

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [x] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` to the beginning of the title and remove the prefix when the PR is ready.
- **User-facing changes**
    - [ ] Not Applicable
    - **OR**
    - [x] User-facing [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed - Issue link:
- **Testing - New and feature**
    - [x] Not Applicable or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
        - [ ] [Add `[feature]` if the change is significant or affects feature tests](https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/testing-guidelines.md#requesting-additional-tests-for-a-pr)

### CODE REVIEW

**Comment by lassehastrup** [comment]
Great add! Also the book that got me started with PowerShell!

**Review by TravisEz13 — APPROVED** [review]

**Comment by TravisEz13** [comment]
There is a duplicate PR:  https://github.com/PowerShell/PowerShell/pull/8041

**Review by SteveL-MSFT — DISMISSED** [review]
LGTM

**Comment by tobvil** [comment]
The PR: #8041 references the 2nd edition of the book. The 3rd is the edition in this PR is the newest edition of the book.

File: docs/learning-powershell/README.md [hunk 1]
```diff
@@ -120,6 +120,8 @@ Commercial Resources
 - [Windows PowerShell in Action][in-action] by Bruce Payette
 - [Introduction to PowerShell][powershell-intro] from Pluralsight
 - [PowerShell Training and Tutorials][lynda-training] from Lynda.com
+- [Learn Windows PowerShell in a Month of Lunches](https://www.amazon.com/Learn-Windows-PowerShell-Month-Lunches/dp/1617294160) by Don Jones and Jeffrey Hicks
```
Comment (adityapatwardhan) [inline docs/learning-powershell/README.md]: I would like the same pattern. Please move the URL below as others.

**Review by adityapatwardhan — CHANGES_REQUESTED** [review]

**Review by adityapatwardhan — APPROVED** [review]

**Pull request merged**